README.adoc

```diff
@@ -0,0 +1,83 @@
+
+[[contributing]]
+Contributing
+~~~~~~~~~~~~
+
+You can develop link:./docs/walkthroughs/local-setup.adoc[locally on your
+host]. Please include as much info as possible in Issues and Pull
+Requests. Merging to master requires approval from a reviewer and a
+passing CI build.
+
+[[communication]]
+Communication
+~~~~~~~~~~~~~
+
+Daily communication happens on #feedhenry on
+https://webchat.freenode.net/[freenode IRC]. The
+http://feedhenry-dev.2363497.n4.nabble.com/[feedhenry-dev@redhat.com
+mailing list] is also used for team-wide & community comms. Issues are
+tracked in both
+https://issues.jboss.org/secure/RapidBoard.jspa?rapidView=4143&view=planning.nodetail[Jira]
+and Github Issues. Where issues are duplicates, they should be linked so
+that only 1 source of info exists (automation would be nice here).
+Typically the core Red Hat team will create and work from Jira Issues.
+
+[[onboarding-resources]]
+Onboarding Resources
+~~~~~~~~~~~~~~~~~~~~
+
+* Mobile SDKs
+** https://github.com/feedhenry/fh-sync-android[Android Sync SDK]
+** https://github.com/feedhenry/fh-sync-js[Cordova/Browser Sync SDK]
+** https://www.aerogear.org/docs/specs/#push[Push SDKs]
+** https://www.npmjs.com/package/keycloak-js[Keycloak JS Adapter]
+* Backend Resources
+** https://tour.golang.org/welcome/1[Tour of Go]
+* Frontend Resources
+** https://github.com/feedhenry/mcp-standalone/tree/master/ui[UI src]
+** https://docs.angularjs.org/tutorial[AngularJS PhoneCat Tutorial]
+** https://docs.angularjs.org/api[AngularJS API Docs]
+** http://www.patternfly.org/[Patternfly]
+** https://github.com/openshift/origin-web-console[OpenShift Web Console]
+** https://docs.openshift.com/container-platform/3.6/install_config/web_console_customization.html[Customising
+the OpenShift Web Console (Extensions]
+** https://github.com/openshift/origin-web-catalog[Service
+Catalog/OpenShift Mall UI]
+* Catalog/Mall, Brokers & Services
+** https://docs.openshift.com/container-platform/3.6/architecture/service_catalog/index.html[Service
+Catalog]
+** https://docs.openshift.com/container-platform/3.6/architecture/service_catalog/ansible_service_broker.html[Ansible
+Service Broker (ASB) & Ansbile Playbook Bundles (APB)]
+** https://docs.openshift.com/container-platform/3.6/architecture/service_catalog/template_service_broker.html[Template
+Service Broker]
+** https://github.com/feedhenry/fh-sync-server[fh-sync-server]
+** https://github.com/feedhenry/fh-sync-server/blob/master/fh-sync-server-DEVELOPMENT.yaml[fh-sync-server
+Template]
+** https://github.com/keycloak/keycloak[Keycloak]
+** https://github.com/feedhenry/keycloak-apb[Keycloak APB]
+
+[[walkthroughs]]
+Walkthroughs
+~~~~~~~~~~~~
+
+* link:./docs/walkthroughs/local-setup.adoc[Local Setup]
+* link:./docs/walkthroughs/developing-apbs-locally.adoc[Developing APBs Locally]
+* link:./docs/walkthroughs/sync-and-keycloak.adoc[Data Sync & Keycloak Walkthough]
+* link:./docs/walkthroughs/mobile-ci-cd.adoc[Mobile CI/CD Walkthough]
```

Comment:
@aidenkeating fyi, added a link to the mobile ci/cd walkthrough here after rebase

Reply:
Ooh good catch 🎣  👍